src/sidebar/websocket.js

```diff
@@ -1,8 +1,19 @@
 import retry from 'retry';
 import EventEmitter from 'tiny-emitter';
 
-// see https://developer.mozilla.org/en-US/docs/Web/API/CloseEvent
-const CLOSE_NORMAL = 1000;
+// Status codes indicating the reason why a WebSocket connection closed.
+// See https://developer.mozilla.org/en-US/docs/Web/API/CloseEvent and
+// https://tools.ietf.org/html/rfc6455#section-7.4.
+
+// "Normal" closures.
+export const CLOSE_NORMAL = 1000;
+export const CLOSE_GOING_AWAY = 1001;
+
+// "Abnormal" closures.
+export const CLOSE_ABNORMAL = 1006;
```

Comment:
Looks like, at this point, only the tests are making use of the specific `CLOSE_ABNORMAL` value here. If all other codes are considered abnormal, perhaps we don't need this reference?

Reply:
> Looks like, at this point, only the tests are making use of the specific CLOSE_ABNORMAL value here. 

You're right, but I thought it would be easier to avoid accidental duplication in future if all of the status codes were defined in the same place.